src/styles/filter.scss

```diff
@@ -506,7 +510,7 @@
 
 	&__label {
 		width: 120px;
-		font-weight: bold;
+		font-weight: 600;
```

Comment:
when font weight is being changed it actually makes the label a bit wider, and it moves the rest of the filter row, can you fix it?